### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by FintanH**
We have moved to Matrix so we're changing the link from IRC to Matrix.
There _is_ something substantial living here now :)

### CODE REVIEW

**Review by lftherios — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -7,9 +7,10 @@ code collaboration protocol and stack.
 
 **🚨 WORK IN PROGRESS 🚨**
 
-_In fact, there is nothing substantial to see here yet_
+While `radicle-link` is currently being used in the `Upstream` application it is
+under heavy development.
```
Comment (kim) [inline README.md:11]:
```suggestion
under heavy development.

Note that [`Upstream`](https://github.com/radicle-dev/radicle-upstream) currently tracks the `maint` branch, while development happens on `next`.
```

File: README.md [hunk 2]
```diff
@@ -7,9 +7,10 @@ code collaboration protocol and stack.
 
 **🚨 WORK IN PROGRESS 🚨**
```
Comment (abbey-titcomb) [inline README.md:8]: Should we remove this now since Radicle Link is ~* live *~ ?

**Review by abbey-titcomb — APPROVED** [review]
LGTM

File: README.md (hunk 2 above)
Comment (FintanH) [reply README.md:8]: I'll remove it when we fully release 😝

**Review by xla — APPROVED** [review]
🍓 🛋 💷 1️⃣

**Comment by FintanH** [comment]
Closed via https://github.com/radicle-dev/radicle-link/commit/39565c7023bdef4c0656a5438b5fc7c57e8720fd

**Pull request closed without merging**